.github/workflows/add_identifiers.yml

```diff
@@ -8,27 +8,39 @@ jobs:
     name: Validate
     uses: ./.github/workflows/validate_secrets.yml
     secrets: inherit
-
+  
```

Comment:
Whitespace added.

Reply:
Sorry about that. I made it identical to LoopWorkspace (which has that extra whitespace.) (validate_secrets, add_identifiers and create_certs match exactly. build_trio/build_loop have the expected differences.)

I propose we leave it for now because we already have 2 approvals and whitespace doesn't matter. If there are future changes to this file, I expect the whitespace get fixed at that time.

If @dnzxy disagrees - I'll change it.

Reply:
In that case just leave it as is.